openreview/venue/matching.py

```diff
@@ -1309,3 +1366,15 @@ def deploy(self, assignment_title, overwrite=False, enable_reviewer_reassignment
         if self.is_reviewer and enable_reviewer_reassignment:
             hash_seed=''.join(random.choices(string.ascii_uppercase + string.digits, k = 8))
             self.setup_invite_assignment(hash_seed=hash_seed, invited_committee_name=f'''Emergency_{self.match_group_name}''')
+
+    def undeploy(self):
+
+        ## Undeploy assignments
+        if self.is_senior_area_chair and not self.venue.sac_paper_assignments:
+            self.undeploy_sac_assignments()
```

Comment:
Has this been implemented already? I can't find it

Reply:
oops, I forgot it!

I think in this case it is only deleting the assignments edges?

Reply:
Correct! About changing the status of the configuration, will this be done in the matcher code?